usaspending_api/references/v2/views/city.py

```diff
@@ -134,11 +160,10 @@ def query_elasticsearch(query, search_text):
 def parse_elasticsearch_response(hits, search_text):
```

Comment:
Again, not your code, but `search_text` doesn't seem to be used.  Should we remove it?

Reply:
It was used before these changes. Removing makes sense

Reply:
Done